res/controllers/Hercules-DJControl-Inpulse-300-script.js

```diff
@@ -0,0 +1,278 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+//////////////////////////////////////////////////////////////////////// 
```

Comment:
Please remove the whole JSHint configuration block. This isn't necessary anymore, were using `eslint` now.